src/tests/conformance/post_handshake/validators.rs

```diff
@@ -55,3 +71,259 @@ async fn c015_TM_VALIDATOR_LIST_COLLECTION_node_should_send_validator_list() {
+    let sig = engine.sign_ecdsa(&message, secret_key);
+    let sigser = sig.serialize_der();
+    let sigb64 = base64::encode(sigser);
+    base64::decode(sigb64).expect("unable to decode a blob")
+}
+
+#[tokio::test]
+async fn c026() {
+    // Create stateful node.
+    let target = TempDir::new().expect("unable to create TempDir");
+    let mut node = Node::builder()
+        .log_to_stdout(true)
+        .start(target.path(), NodeType::Stateless)
+        .await
+        .expect("unable to start stateful node");
+
+    let mut test_config = TestConfig::default();
+    test_config.synth_node_config.generate_new_keys = false;
+    let synth_node = SyntheticNode::new(&test_config).await;
+
+    synth_node
+        .connect(node.addr())
+        .await
+        .expect("unable to connect");
+
+    // 1. Setup keys & prefix.  Both master and signing key pairs have been previously generated.
+    let master_secret_hex =
+        String::from("8484781AE8EEB87D8A5AA38483B5CBBCCE6AD66B4185BB193DDDFAD5C1F4FC06");
+    // The master public key should be in the validators.txt file, in ~/.ziggurat/ripple/setup
+    let master_public_hex =
+        String::from("02ED521B8124454DD5B7769C813BD40E8D36E134DD51ACED873B49E165327F6DF2");
+    let master_secret_bytes = hex::decode(&master_secret_hex).expect("unable to decode hex");
+    let master_public_bytes = hex::decode(&master_public_hex).expect("unable to decode hex");
+    let master_secret_key =
+        SecretKey::from_slice(master_secret_bytes.as_slice()).expect("unable to create secret key");
+
+    let signing_secret_hex =
+        String::from("00F963180681C0D1D51D1128096B8FF8668AFDC41CBDED511D12D390105EFDDC");
+    let signing_public_hex =
+        String::from("03859B76317C8AA64F2D253D3547831E413F2663AE2568F7A17E85B283CC8861E4");
+    let signing_secret_bytes = hex::decode(&signing_secret_hex).expect("unable to decode hex");
+    let signing_public_bytes = hex::decode(&signing_public_hex).expect("unable to decode hex");
+    let signing_secret_key = SecretKey::from_slice(signing_secret_bytes.as_slice())
+        .expect("unable to create secret key");
+    let man_prefix: Vec<u8> = vec![b'M', b'A', b'N', 0];
+
+    // 2. Create signable manifest with sequence, public key, signing public key (without signatures)
+    if master_public_bytes.len() != PUBLIC_KEY_SIZE {
+        panic!(
+            "invalid master public key length: {}",
+            master_public_bytes.len()
+        );
+    }
+    if signing_public_bytes.len() != PUBLIC_KEY_SIZE {
+        panic!(
+            "invalid signing public key length: {}",
+            signing_public_bytes.len()
+        );
+    }
+    let signable_manifest =
+        create_signable_manifest(1, &master_public_bytes, &signing_public_bytes);
+
+    // 3. append manifest prefix
+    let mut prefixed_signable: Vec<u8> = vec![0; signable_manifest.len() + 4];
+    prefixed_signable[0..4].clone_from_slice(man_prefix.as_slice());
```

Comment:
Would maybe this do the trick? Then the `man_prefix` variable is not needed at all.
```suggestion
    prefixed_signable[0..4].clone_from_slice(b"MAN\x00");
```